### ISSUE TITLE
Extrude for rectangles

### ISSUE THREAD

**Issue by mohammedzamakhan** [uber/nebula.gl#291]
#### Feature request
For rectangular shapes (Rectangle/RectangleUsing3Points), when I edit using extrude, I want to be able to just use one point to change the width and height. Currently I have to use two different points to do it.

#### To Do List 
  - [ ] Add label and assign to milestone
  - [ ] Coding
  - [ ] Doc update
  - [ ] Test

**Comment by supersonicclay** [comment]
If you are in Extrude mode, you can drag the edge to move both points. Does this accomplish what you're after?

![rectangle-extrude](https://user-images.githubusercontent.com/469582/67505322-e103b580-f63f-11e9-901f-ad84427d74b4.gif)

**Comment by mohammedzamakhan** [comment]
Yes I know you can use Extrude to do it, but its a two-step process (width and height), I just want to select one edge and drag to change height and width together, rather than a two-step process

**Comment by supersonicclay** [comment]
Ah, I see what you're saying. That'd be an interesting enhancement to the extrude mode. It'd need to handle polygons that aren't quite rectangles also since GeoJSON rectangles are not represented any differently than any other quadrilateral.
